core/backend/src/test/standalone/TileCache.test.ts

```diff
@@ -49,7 +49,7 @@ export async function getTileProps(iModel: IModelDb): Promise<TileContentRequest
       continue;
 
     iModelTileTreeIdToString;
-    const treeId = iModelTileTreeIdToString(modelId, { type: BatchType.Primary, edges: EdgeType.None }, defaultTileOptions);
+    const treeId = iModelTileTreeIdToString(modelId, { type: BatchType.Primary, edges: false as const }, defaultTileOptions);
```

Comment:
For my edification: what does as `false as const` do?

Reply:
If you omit `as const`, compiler infers type `boolean` and whines that `boolean` is not compatible with `EdgeOptions | false`. `as const` tells it not to widen the inferred type from `false` to `boolean`.